pallets/collator-assignment/src/mock.rs

```diff
@@ -284,7 +291,10 @@ pub struct MockCollatorAssignmentTip;
 
 impl CollatorAssignmentTip<u32> for MockCollatorAssignmentTip {
     fn get_para_tip(para_id: ParaId) -> Option<u32> {
-        if MockData::mock().apply_tip && (para_id == 1003u32.into() || para_id == 1004u32.into()) {
+        if MockData::mock().apply_tip
+            && ((para_id == 1003u32.into() || para_id == 1004u32.into())
+                || MockData::mock().chains_that_are_tipping.contains(&para_id))
```

Comment:
You can initialize `chains_that_are_tipping` to `vec![1003, 1004]`, and simplify this to `if apply_tip && chains_that_are_tipping.contains`.

Reply:
*fixed*